Code/src/test/java/nl/utwente/group10/ui/components/blocks/FunctionBlockTest.java

```diff
@@ -20,9 +20,10 @@
      */
     @Before
     public void setUp() throws Exception {
+        //TODO: not entirely correct.
```

Comment:
That is not in our catalog :(
I wanted to update this along with the other type changes